_episodes/02-running-containers.md

```diff
@@ -180,12 +180,29 @@ with Singularity available.
 > {: .solution}
 {: .challenge}
 
-## Exiting a singularity image
+## Exiting a Singularity image
 
-The `exit` command exits a singularity instance. Note that when exiting from the singularity image all the running processes are killed (stopped).
+The `exit` command exits a Singularity instance. Note that when exiting from the Singularity image all the running processes are killed (stopped).
 
 ```bash
 Singularity> exit
 ```
 
+## Clean Singularity cache
+
+To remove Singularity image from the cache memory use:
```

Comment:
Missing "a" Singularity image... Also comma before "use". Also I'm confused by the word "memory", because this usually means RAM. Maybe just delete "memory" if that's not what you mean?

Reply:
Maybe it'd be best to actually explain what the "cache" is (that'd probably also answer my question).

Reply:
Thanks for this useful comment. 
​
No I didn't mean RAM.

​So when an image is downloaded, it is stored in a cache ​folder ($HOME/.singularity/cache). So the ​`... cache clean ...` command removes the image from cache​ folder​.

I think ​we **need to check whether** `singularity delete` and `singularity cache clean` commands are similar. (This we might also need to specify in the module.)?

Reply:
For the moment I am removing the information on cache. lets discuss this in a new issue.